crates/oct-py/python/opencloudtool/py_api.py

```
@@ -64,3 +64,78 @@ def destroy(path: str = "."):
+        reqs_path = os.path.join(project_root, "requirements.txt")
+        with open(reqs_path, "w") as f:
+            f.write(requirements_content)
+        logger.info(f"Generated requirements.txt at '{reqs_path}'")
+
+        toml_path = os.path.join(project_root, "oct.toml")
+        with open(toml_path, "w") as f:
+            f.write(oct_toml_content)
+        logger.info(f"Generated oct.toml at '{toml_path}'")
+
+    except IOError as e:
+        logger.error(f"Error writing configuration files: {e}")
```

Ruff (0.13.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)